extensions/amp-user-notification/0.1/amp-user-notification.css

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2015 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+amp-user-notification {
+  position: fixed !important;
+  top: 0;
+  left: 0;
+  overflow: hidden !important;
+  background: rgba(255, 255, 255, .7);
+  z-index: 1;
```

Comment:
we should consider creating a file for maintaining z-indices and layers and use constants here, else this is going to get unmanageable soon. May be open a ticket for this i shall look into that.